gradle/libs.versions.toml

```diff
@@ -28,6 +28,7 @@ kotlin-multiplatform = { id = "org.jetbrains.kotlin.multiplatform", version.ref
 kotlin-native-cocoapods = { id = "org.jetbrains.kotlin.native.cocoapods", version.ref = "kotlin" }
 serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin" }
 ktlint = { id = "org.jlleitschuh.gradle.ktlint", version.ref = "ktlint" }
+owasp-dependencycheck = { id = "org.owasp.dependencycheck", version = "8.4.0" }
```

Comment:
Let's place it in alphabetical order (above `serialization`) to keep it constant 🙏🏻

Reply:
It was already not in alphabetical order, afaik?

`serialization` being before `ktlint`... just sayin' 😏

Reply:
You're absolutely right 🤦‍♂️ Let's fix ktlint too then 😅